rodi/utils/class_name.py

```diff
@@ -0,0 +1,11 @@
+def class_name(input_type):
+    if input_type in {list, set} and str(
+        type(input_type) == "<class 'types.GenericAlias'>"
+    ):
```

Comment:
```suggestion
    generic_alias = "<class 'types.GenericAlias'>"
    if input_type in {list, set} and str(type(input_type)) == generic_alias:
```
However.. this is unrelated to this issue, but it doesn't works as expected